factfinder/main.py

```diff
@@ -436,9 +439,17 @@ def aggregate_horizontal(self, v: Variable, geotype: str) -> pd.DataFrame:
         # Aggregate variables horizontally
         df["pff_variable"] = v.pff_variable
         df["geotype"] = geotype
-        df["e"] = df[E_variables].sum(axis=1)
+
+        df["e"] = (
+            df[E_variables].sum(axis=1)
+            # Special Case for WrkrNotHm = DP03_0018E - DP03_0024E
+            if v.pff_variable != "wrkrnothm"
+            else df.DP03_0018E - df.DP03_0024E
```

Comment:
Is it possible that we will have other examples of subtraction in the future?

Reply:
maybe, but not yet. in these cases, we should probably just name the two census variables -> `DP03_0018E` and `DP03_0024E` as acs_variables, then add `wrkrnothm` as a special variable that depend on `DP03_0018E` and `DP03_0024E`

Reply:
oh wait they are actual variables, 
wrkr16pl = DP03_0018
cw_wrkdhm = DP03_0024, 

we should just add wrkrnothm as a special variable then